### PR TITLE
feat(B-589): Workflow in GH Actions for semaphore-airdrop-relayer

### PR DESCRIPTION
**Pull request by jazzbee**
Workflow to build and deploy application.

### CODE REVIEW

File: chart/templates/hpa.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```
Comment (kszarek) [inline chart/templates/hpa.yaml:1]: Do we want to enable it?

File: .github/workflows/ecr-build-push.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          repository: worldcoin/gh-actions
+          ref: v2
+          token: ${{ secrets.GIT_HUB_TOKEN }}
+          path: .github/actions/gh-actions
+      - name: Build & Push to ECR
+        uses: ./.github/actions/gh-actions/ecr-build-and-push
+        with:
+          ecr_repo: ${{ inputs.ecr_repo }}
+          aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
```
Comment (kszarek) [inline .github/workflows/ecr-build-push.yml:34]: can we use OIDC role instead of getting AWS creds from Github credentials?

Comment (jazzbee) [reply .github/workflows/ecr-build-push.yml:34]: We use OIDC ... this is not a credential... this is just an AWS Account ID.

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# Default values for semaphore-airdrop-relayer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+awsAccountID: "267301337261"
```
Comment (kszarek) [inline chart/values.yaml:7]: we keep it already in the secrets, no need to have it here as well

File: chart/templates/ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                name: {{ $fullName }}
+                port:
+                  number: {{ $svcPort }}
+              {{- else }}
+              serviceName: {{ $fullName }}
+              servicePort: {{ $svcPort }}
+              {{- end }}
+          {{- end }}
+    {{- end }}
+{{- end }}
```
Comment (kszarek) [inline chart/templates/ingress.yaml:61]: cool, can we remove it and add a readable version?

File: chart/values.yaml (hunk 3 above)
Comment (jazzbee) [reply chart/values.yaml:7]: 👍

Comment (jazzbee) [reply chart/values.yaml]: https://github.com/worldcoin/semaphore-airdrop-relayer/pull/6/commits/817b00da0660d72d85c6b74de4d2ff0779ed201e

File: chart/templates/hpa.yaml (hunk 1 above)
Comment (jazzbee) [reply chart/templates/hpa.yaml]: https://github.com/worldcoin/semaphore-airdrop-relayer/pull/6/commits/55fe421b321e8e714264be3042fe9429832a0f42

File: chart/templates/ingress.yaml (hunk 4 above)
Comment (jazzbee) [reply chart/templates/ingress.yaml]: https://github.com/worldcoin/semaphore-airdrop-relayer/pull/6/commits/94bf02e36aa497574e29ad883564dc3ffdcef3d7

**Review by jmarmuszewski — APPROVED** [review]

**Pull request merged**